docker-compose.dev.yaml

```diff
@@ -8,6 +8,7 @@ services:
       - "${SERVER_PORT:-8000}:8000" # Map server port on the host to port 8000 in the container
     volumes:
       - ./tronbyt_server:/app/tronbyt_server # for development
+      - ./users:/app/users # for development
```

Comment:
Was this intended?

Reply:
yeah. without it the old usersdb file won't be picked up from the local file system.